README.md

```diff
@@ -1 +1,5 @@
-# Review-past-project
\ No newline at end of file
+# Review-past-project
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
I think it would be better to make a more descriptive readme file for your project, it would enable users to understand what the project is presenting and make your project look more professional